controllers/management.js

```diff
@@ -0,0 +1,14 @@
+const { removeHostById } = require('../services/fleet');
+
+exports.removeHost = async (req, res) => {
+    try {
+        const hostId = req.params.id;
+        const hostInfo = req.body; // Ensure the client sends the necessary host info in the request body
+        console.log(`Received request to remove host with ID: ${hostId}`);
+        const result = await removeHostById(hostId, hostInfo);
+        res.status(200).json({ message: 'Host removed successfully', data: result });
```

Comment:
Should respond with "task sent successfully" for the request not to take such a long time to answer. In order to get response of success/failure another feature should be implemented, perhaps notifications. Look at some of Mazi's functions for reference